.jshintrc

```diff
@@ -23,7 +23,7 @@
   "undef"         : true,     // true: Require all non-global variables to be declared (prevents global leaks)
   "unused"        : false,    // true: Require all defined variables be used
   "strict"        : false,    // true: Requires all functions run in ES5 Strict Mode
-  "maxparams"     : 10,       // {int} Max number of formal params allowed per function
+  "maxparams"     : 12,       // {int} Max number of formal params allowed per function
```

Comment:
I'm in agreement with @AparnaKarve, we shouldn't be bending the fils for a change.